lib/server-starter.js

```diff
@@ -90,7 +92,23 @@ class Server extends EventEmitter {
     });
 
     // Should be resolved by the "spawn" event, but that is only supported in Node 15+
-    return Promise.resolve;
+    if (process.platform !== 'win32') return Promise.resolve;
+    return new Promise((resolve) => {
```

Comment:
There is no reject handler. What's supposed to happen when it times out?

Reply:
the caller will do exactly the same (trying to connect), and will fail because the port is not ready.

Reply:
A promise does not resolve on its own.